index.html

```diff
@@ -567,6 +581,19 @@ <h3>Productions</h3>
                   </ul>
               </td>
           </tr>
+          <tr>
+              <td>GET</td>
+              <td>/partner/v1/productions/byVendorProductionId/&lt;vendorProductionId&gt;</td>
+              <td><a href="#AtomProductionForVendorProductionExample">Get Atom Production for VendorProductionId</a></td>
```

Comment:
Get the production details by Vendor Id

Reply:
i wanted to be specific about atom production vs vendor production